### PR TITLE
FLUID-5561: Convert Pager CSS to Stylus

### PR DESCRIPTION
**Pull request by dinukadesilva**
Removed the css file in pager and replace with a new stylus file.

### CODE REVIEW

**Comment by jobara** [comment]
@Dinuka2013513  thank you for your pull request. We ask that all our contributors sign a CLA. Could you please fill out the Fluid CLA and send it to me. https://wiki.fluidproject.org/display/fluid/Fluid+Licensing You can ping me in the IRC channel (Justin_o) if you have more questions on that.

For background reading, here's a good overview of what CLAs are and why they're important: http://oss-watch.ac.uk/resources/cla

**Comment by jobara** [comment]
@Dinuka2013513 thanks for sending me the CLA

File: src/components/pager/css/stylus/Pager.styl [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+.fl-theme-bw .fl-pager .fl-pager-data {
+    .fl-pager-desc {
+        background: url(../images/arrow-up.png) no-repeat right;
+    }
+    .fl-pager-asc {
+        background: url(../images/arrow-dn.png) no-repeat right;
+    }
+}
+
+/* Pager UI */
```
Comment (cindyli) [inline src/components/pager/css/stylus/Pager.styl]: This comment can be remove.

Comment (Dinuka2013513) [inline src/components/pager/css/stylus/Pager.styl]: removed

**Comment by cindyli** [comment]
Thanks for the fix, @Dinuka2013513, could you add the ticket number, in your case FLUID-5561, to all commit messages for the future reference of knowing the purpose of each commit. 

When I was testing this pull request, I realized some stylings in the Pager.css file are no longer valid, such as at this line - https://github.com/fluid-project/infusion/blob/master/src/components/pager/css/Pager.css#L20, the selector ```fl-pager-ui``` that is meant to be an umbrella selector for the pager UI doesn't appear in any demo or example markups or javascript code. Most, or probably all, pager styling on the demo (https://github.com/fluid-project/infusion/blob/master/demos/pager/index.html) comes from the demo css file, in this demo - https://github.com/fluid-project/infusion/blob/master/demos/pager/css/pager.css. There are other pager css files for other pager examples in this repository. You can search the repo using ```pager.css```.

This brings some thoughts of extending the coverage of this JIRA by revisiting the pager css contents, including css files for the Pager component, demos and examples, to:

1. Remove the parts in the component Pager.css that are no longer valid;
2. Better make use the css in the demo (or example) pager css files by moving the sensible pieces into the component Pager css file;
3. Of course, the new component Pager.css is written in stylus.

Let me know what you think. Thanks.

**Comment by cindyli** [comment]
@Dinuka2013513, thanks for removing css that are no longer in use. 

However, the goal of this updated JIRA is to consolidate Pager css that are from various places: 
1) the component css, which is the one you've been working on in this pull request; 
2) css from different Pager demos

It would be great if you could analyze and play with these css files to collect useful parts into the component css file so that every single Pager integration don't have to repeat the same css.

**Comment by cindyli** [comment]
@Dinuka2013513, actually another pull request https://github.com/fluid-project/infusion/pull/685/ is modifying and consolidating the pager component and demo css files. The stylus conversion should happen after that when we have a clean component css.

**Comment by cindyli** [comment]
@Dinuka2013513, do you plan to continue working on this pull request? If not, this pull request will be closed due to its inactivity.

**Comment by Dinuka2013513** [comment]
@cindyli, actually I have completed everything related to this pull request as of now. But, shouldn't I wait for the #685 since it carries some more changes related to this pull request.

**Comment by amb26** [comment]
Closed due to inactivity

**Pull request closed without merging**